### PR TITLE
Fix sample schem and health

### PR DESCRIPTION
**Pull request by blabadi**
**Description of changes**

**Checklist before requesting review:**

- [ ] Check branch (code change PRs go to `develop` not master)
- [ ] Manual testing
- [ ] Regression tests completed and passing (double check number of tests).
- [ ] Spelling has been checked.
- [ ] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

File: src/bootstrap.ts [hunk 1]
```diff
@@ -176,6 +179,11 @@ export const run = async (config: AppConfig) => {
       throw new Error('App is not configured correctly either provide jwt pub key url or key');
     }
     setJwtPublicKey(config.jwtPubKeyUrl());
+    setInterval(() => {
```
Comment (jaserud) [inline src/bootstrap.ts:182]: does the public key change every 5min now??

Comment (jaserud) [inline src/bootstrap.ts:182]: does the public key change every 5min now??

File: src/bootstrap.ts [hunk 2]
```diff
@@ -123,22 +124,24 @@ const setupRxNormConnection = (conf: RxNormDbConfig) => {
     timeout: conf.timeout,
   });
 
-  pool.on('error', err => setStatus('rxNormDb', { status: Status.ERROR }));
   pool.on('connection', () => setStatus('rxNormDb', { status: Status.OK }));
+
+  setStatus('rxNormDb', { status: Status.OK });
```
Comment (jaserud) [inline src/bootstrap.ts:129]: won't this always be status `OK` now?? I don't see anywhere it would be set to error.

File: src/bootstrap.ts (hunk 1 above)
Comment (blabadi) [reply src/bootstrap.ts:182]: No this is just something should been done before to adapt automatically without manual restarts

Comment (jaserud) [reply src/bootstrap.ts:182]: I see makes sense

File: src/bootstrap.ts (hunk 2 above)
Comment (blabadi) [reply src/bootstrap.ts:129]: yep, it was only checking on the beginning I tried set interval and ping the db but that didn't work.. that check missed up with the tests somehow. so for now I just want to get done with it, not urgent thing

Comment (jaserud) [reply src/bootstrap.ts:129]: cool as long as we don't forget it.

**Comment by blabadi** [comment]
> Looks like some tests fail eh. Running locally i got errors with rx db

yeah tests are still failing even without the set interval, I'm checking again, it's weird
I think something related to test container

**Comment by blabadi** [comment]
turned out to be the wait on log message was no good, it tried to connect too early, should pass now

**Pull request merged**